services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentService.kt

```diff
@@ -33,7 +35,54 @@ class ApplicationAttachmentService(
     private val applicationRepository: ApplicationRepository,
     private val attachmentContentService: ApplicationAttachmentContentService,
     private val scanClient: FileScanClient,
+    private val lockService: LockService,
+    private val contentTransferService: ApplicationAttachmentContentTransferService,
 ) {
+
+    internal val lockName = "applicationAttachmentBlobTransfer"
+
+    @Scheduled(
+        fixedDelayString = "\${haitaton.application.attachments.blobTransferIntervalMilliSeconds}",
+        initialDelayString =
+            "\${haitaton.application.attachments.blobTransferInitialDelayMilliSeconds}"
+    )
+    fun transferAttachmentContentToBlobStorage() {
```

Comment:
I think this should be in a separate service, conditional on an environment variable so we can turn the scheduled method off when it's completed.

Maybe similar to the hanke attachment migration scheduler: https://github.com/City-of-Helsinki/haitaton-backend/commit/40dcc7dac5e54c4a8086722c97c07debcb1bbd44

Reply:
Oh, I wish I had thought to look into that Hanke attachment migration. But, I will do it now.

Reply:
I refactored the whole thing to be more like what was done with Hanke attachments.